### DISCUSSION TITLE
Veggie Details View

### DISCUSSION THREAD

**Discussion by InMatrix** [InMatrix/veggieseasons_adaptive#2] [General]
Here's a side-by-side comparison of the Veggie Details View between the M3 version of Veggie Seasons (left) and the original (right). The M3 version was built with [showModalBottomSheet](https://api.flutter.dev/flutter/material/showModalBottomSheet.html), while the original was built with `Row` and `Column` widgets and shown via a `CupertinoPageRoute`.

- What in the M3 version stands out to you from an iOS app design's perspective? 
- What, if any, you think the M3 version _must_ adapt to iOS to make this screen look natural on iOS?
- If you can spend more time refining this page for iOS users, what else would you change while maintaining a largely shared codebase across Android and iOS? 

<img width="700" alt="image" src="https://user-images.githubusercontent.com/348942/221037611-7f93114c-5bd7-40ff-81a3-448c1bac119c.png">

<img width="698" alt="image" src="https://user-images.githubusercontent.com/348942/221039593-25fbb682-72ba-48d4-b548-2133295d7350.png">

**Comment by nelmerdlc** [comment]
The tabs/toggle(?) between trivia and info is really good. To be honest I want to move into a world where product folks continue to create these "platform agnostic" UI.
